cmd/minikube/cmd/config/disable_test.go

```diff
@@ -16,74 +16,10 @@ limitations under the License.
 
 package config
 
-import (
-	"io/ioutil"
-	"os"
-	"path/filepath"
-	"strings"
-	"testing"
-
-	"github.com/docker/machine/libmachine/drivers"
-
-	"k8s.io/minikube/pkg/minikube/assets"
-	"k8s.io/minikube/pkg/minikube/sshutil"
-	"k8s.io/minikube/pkg/minikube/tests"
-)
+import "testing"
 
 func TestDisableUnknownAddon(t *testing.T) {
 	if err := Set("InvalidAddon", "false"); err == nil {
 		t.Fatalf("Disable did not return error for unknown addon")
 	}
 }
-
-func TestDisableValidAddonLocal(t *testing.T) {
```

Comment:
No, a pretty big part of theses tests were checking if the file itself was transferred, which should be covered by the SSHRunner tests.  

There's probably a bit of coverage that got lost about writing the config and making sure that the addon gets configured properly.